### PR TITLE
Fixes #140

### LINKED ISSUE #140: Blog posts with code blocks get cut off on mobile devices
### Before you start...

- [X] Have you refreshed the page? You might be using an old version of the site.
- [X] Have you checked for an existing issue on this topic? If there is one, post a comment on it instead.

### What browsers are you seeing the problem on?

Chrome

### Description

The blog post ["How to use a custom background in Files"](https://files.community/blog/posts/how-to-use-background-image) does not automatically adjust to the width of small screen mobile devices. 

### Steps To Reproduce

Open the [website](https://files.community/blog/posts/how-to-use-background-image) in a mobile device. 

### Expected behavior

Like the other blog posts, the custom background blog should also be easily readable on mobile devices. The text width should automatically fit smaller screens. 

### Relevant Assets

image

### PR DESCRIPTION
**Pull request by babakfp**
Screenshot 2022-05-16 211844

### CODE REVIEW

File: src/styles/pages/blogPost.scss [hunk 1]
```diff
@@ -1,8 +1,10 @@
 @use "../mixins" as *;
 
 .blog-post {
-	@include flex($justify: center);
 	padding: 48px;
+  margin-right: auto;
+  margin-left: auto;
```
Comment (TheOnlyTails) [inline src/styles/pages/blogPost.scss:6]:
```suggestion
  margin-inline: auto;
```

File: src/styles/pages/blogPost.scss [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+  max-width: 100%;
```
Comment (TheOnlyTails) [inline src/styles/pages/blogPost.scss:7]:
```suggestion
  max-inline-size: 100%;
```

**Review by TheOnlyTails — CHANGES_REQUESTED** [review]
Looks interesting, just use logical units.

**Review by TheOnlyTails — COMMENTED** [review]
Looks fine, but I did notice there is a small horizontal scrolling. Could you fix that?

**Comment by babakfp** [comment]
> Looks fine, but I did notice there is a small horizontal scrolling. Could you fix that?

I didn't find any "horizontal scrolling". Maybe it's caused by the browser itself, sometimes chromes responsive view doesn't work properly and it shows a horizontal scrollbar🤷‍♂️.

**Comment by babakfp** [comment]
Never mind, I find it. It just doesn't appear in this post ["How to use a custom background in Files"](https://files.community/blog/posts/how-to-use-background-image).

**Review by TheOnlyTails — APPROVED** [review]
lgtm!

**Pull request merged**